cli/tools/installer.rs

```diff
@@ -51,7 +51,10 @@ fn generate_executable_file(
   mut file_path: PathBuf,
   args: Vec<String>,
 ) -> Result<(), AnyError> {
-  let args: Vec<String> = args.iter().map(|c| format!("\"{}\"", c)).collect();
+  let args: Vec<String> = args
```

Comment:
These args are nit just used by the generated .cmd script, but also the generated script compatible with git bash / MINGW64. (On windows deno install creates two scripts). Should update this to only apply the escaping to the .cmd script.